src/__test__/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.test.jsx

```diff
@@ -275,4 +286,44 @@ describe('Marker heatmap plot', () => {
     // The list of displayed genes should be in the same order as the displayed genes
     expect(_.isEqual(genesListAfterRemoval, genesListBeforeRemoval)).toEqual(true);
   });
+
+  it('loads the tabs under gene selection', async () => {
+    await renderHeatmapPage(storeState);
+
+    expect(screen.getByText(/Add\/Remove genes/i)).toBeInTheDocument();
+    expect(screen.getByText(/Re-order genes/i)).toBeInTheDocument();
+  });
+
+  it('switches tabs, re-orders and removes genes using the tree', async () => {
+    await renderHeatmapPage(storeState);
+
+    await act(async () => {
+      userEvent.click(screen.getByText('Re-order genes'));
+    });
+
+    // Check that initially there are 5 marker genes - the default
+    markerGenesData5.order.forEach((geneName) => {
+      expect(screen.getByText(geneName)).toBeInTheDocument();
+    });
+
+    const geneTree = screen.getByRole('tree');
+
+    // Introduce testing for drag and drop if possible
+    // In Jest (jsdom) DragEvent and properties to do with rendering (widths, coordinates)
+    // don't exist -> not possible to test?
```

Comment:
If you want to test it one option could be to get the `onDrop` prop your `HierarchicalTreeGenes` component passed to `Tree` and then trigger it yourself. 

```
   const tree = component.find('HierarchicalTreeGenes Tree');
    tree.getElement().props.onDrop(dropInfo);
    component.update();
```

It would involve using enzyme (which kind of breaks a little bit the idea/philosophy of using rtl, but since rtl doesn't support drag and drop I think it makes sense in this case).

This is just and idea because I read this comment, but it's not necessarily something you need to do.